src/coprocessor/dag/executor/mod.rs

```diff
@@ -387,110 +385,22 @@ pub mod tests {
-                for &(ref key, ref value) in kv_data {
-                    if pk.is_empty() {
-                        pk = key.clone();
-                    }
-                    txn.prewrite(
-                        Mutation::Put((Key::from_raw(key), value.to_vec())),
-                        &pk,
-                        &Options::default(),
-                    )
-                    .unwrap();
-                }
-                txn.into_modifies()
-            };
-            self.write_modifies(txn_motifies);
-
-            // do commit
-            let txn_modifies = {
-                let mut txn = MvccTxn::new(self.snapshot.clone(), START_TS, true).unwrap();
-                for &(ref key, _) in kv_data {
-                    txn.commit(Key::from_raw(key), COMMIT_TS).unwrap();
-                }
-                txn.into_modifies()
-            };
-            self.write_modifies(txn_modifies);
-        }
-
-        #[inline]
-        fn write_modifies(&mut self, txn: Vec<Modify>) {
-            self.engine.write(&self.ctx, txn).unwrap();
-            self.snapshot = self.engine.snapshot(&self.ctx).unwrap()
-        }
-
-        pub fn get_snapshot(&mut self) -> (RocksSnapshot, u64) {
-            (self.snapshot.clone(), COMMIT_TS + 1)
-        }
-    }
-
-    pub fn get_range(table_id: i64, start: i64, end: i64) -> KeyRange {
-        let mut key_range = KeyRange::default();
-        key_range.set_start(table::encode_row_key(table_id, start));
-        key_range.set_end(table::encode_row_key(table_id, end));
-        key_range
-    }
-
     pub fn get_point_range(table_id: i64, handle: i64) -> KeyRange {
         let start_key = table::encode_row_key(table_id, handle);
         let mut end = start_key.clone();
         crate::coprocessor::util::convert_to_prefix_next(&mut end);
-        let mut key_range = KeyRange::default();
+        let mut key_range = KeyRange::new();
         key_range.set_start(start_key);
         key_range.set_end(end);
         key_range
     }
 
-    pub fn gen_table_scan_executor(
-        tid: i64,
-        cis: Vec<ColumnInfo>,
-        raw_data: &[Vec<Datum>],
-        key_ranges: Option<Vec<KeyRange>>,
-    ) -> Box<dyn Executor> {
-        let table_data = gen_table_data(tid, &cis, raw_data);
-        let mut test_store = TestStore::new(&table_data);
-
-        let mut table_scan = TableScan::default();
-        table_scan.set_table_id(tid);
-        table_scan.set_columns(cis.clone().into());
-
-        let key_ranges = key_ranges.unwrap_or_else(|| vec![get_range(tid, 0, i64::max_value())]);
-
-        let (snapshot, start_ts) = test_store.get_snapshot();
-        let store = SnapshotStore::new(snapshot, start_ts, IsolationLevel::SI, true);
-        Box::new(TableScanExecutor::table_scan(table_scan, key_ranges, store, false).unwrap())
+    #[inline]
+    pub fn get_range(table_id: i64, start: i64, end: i64) -> KeyRange {
+        let mut key_range = KeyRange::default();
```

Comment:
Changing `KeyRange::default` to `KeyRange::new` in L392 (https://github.com/tikv/tikv/pull/5143/files#diff-afaa9539aadbecbcf6c9af032e38da84R392), but use `KeyRange::default` here, is there special usage?

Reply:
Sorry this should be a merge mistake. It should always be `::default()`. See https://github.com/tikv/tikv/commit/f5215d631d3145d824c0d0940cd490ff3f8bbf80